passenger-service/src/main/java/by/sergo/passengerservice/controller/PassengerController.java

```diff
@@ -0,0 +1,53 @@
+package by.sergo.passengerservice.controller;
+
+import by.sergo.passengerservice.domain.dto.request.PassengerCreateUpdateRequest;
+import by.sergo.passengerservice.domain.dto.response.PassengerListResponse;
+import by.sergo.passengerservice.domain.dto.response.PassengerResponse;
+import by.sergo.passengerservice.service.impl.PassengerServiceImpl;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/api/v1/passengers")
+@RequiredArgsConstructor
+public class PassengerController {
+
+    private final PassengerServiceImpl passengerServiceImpl;
+
+    @PostMapping
+    @ResponseStatus(HttpStatus.CREATED)
+    public ResponseEntity<PassengerResponse> create(@RequestBody @Valid PassengerCreateUpdateRequest dto) {
+        return ResponseEntity.status(HttpStatus.CREATED).body(passengerServiceImpl.create(dto));
```

Comment:
Вот как это должно быть отформатировано, поменяй, чтобы везде так было
```suggestion
        return ResponseEntity.status(HttpStatus.CREATED)
                             .body(passengerServiceImpl.create(dto));
```